client/my-sites/post-share/style.scss

```diff
@@ -1,177 +1,278 @@
 $section-border: solid 1px transparentize( lighten( $gray, 20% ), .5 );
 
-.posts__list .post-share__wrapper {
-	margin: 0;
-	animation: appear .3s ease-in-out;
+.post-share {
 	border-top: $section-border;
 
-	.post-share__title {
-		margin: 24px 0 0 0;
-		font-size: 18px;
-	}
+	.post-share__service.is-active,
```

Comment:
Do we need to nest this? Can't we just add it to top level css like:
```
.post-share {
...
}
.post-share__service.is-active,
.post-share__handle {
...
}
```

The same goes for all the other styles like this.